Sources/WalletSdk/StorageManager.swift

```diff
@@ -141,6 +128,4 @@ class StorageManager: NSObject {
    }
 }
 
-//
 // Copyright © 2024, Spruce Systems, Inc.
```

Comment:
Could you remove those copyrights? I think it's theoretically fine with the MIT/Apache licenses, but it's not something we do

Reply:
Done.